### DISCUSSION TITLE
Is Vercel down? My sites aren't loading for days now

### DISCUSSION THREAD

**Discussion by k16e** [vercel/community#2666] [Help]
### Summary

Hi, it's worrisome that some of my sites are not loading, down for close to 48 hours now of constant checks

k16e.co
cfood.studio
alforghm.com

Is this where to report this kind of incidence?

### Steps to Reproduce

...

**Comment by leerob** [comment]
Hey there, I am not able to reproduce - those sites are loading for me.

Where are you trying to connect from (IP address)? What are you seeing?

https://downforeveryoneorjustme.com/k16e.co

**Comment by k16e** [comment]
This is weird still down for me, won't resolve at all at IP 102.88.34.252


--

Kabolobari Benakole /kb

Founder, K16E, Inc.

​

https://k16e.co











---- On Wed, 31 May 2023 03:16:10 +0100 Lee Robinson ***@***.***> wrote ---





Hey there, I am not able to reproduce - those sites are loading for me.

Where are you trying to connect from (IP address)? What are you seeing?

https://downforeveryoneorjustme.com/k16e.co

**Comment by k16e** [comment]
How is this happening? I set my VPN to "United States" and then all the sites are loading. I don't remember IP-restricting any of my sites/domains. How's it happening automatic? Is it some bug/error from Vercel? It's strange, has not happened before. 


--

Kabolobari Benakole /kb

Founder, K16E, Inc.

​

https://k16e.co











---- On Wed, 31 May 2023 03:16:10 +0100 Lee Robinson ***@***.***> wrote ---





Hey there, I am not able to reproduce - those sites are loading for me.

Where are you trying to connect from (IP address)? What are you seeing?

https://downforeveryoneorjustme.com/k16e.co
